planb-cassandra.sh

```diff
@@ -65,11 +65,15 @@ echo $KEYSTORE | base64 -d > /etc/cassandra/keystore
 echo "Finished bootstrapping node."
 # Add route 53record seed1.${CLUSTER_NAME}.domain.tld ?
 
+ncores=$(grep -c ^processor /proc/cpuinfo)
 #
 # Assuming we are using SSD storage, set memtable_flush_writers to the
 # number of CPU cores:
 #
-export MEMTABLE_FLUSH_WRITERS=$(grep -c ^processor /proc/cpuinfo)
+export MEMTABLE_FLUSH_WRITERS=$ncores
+
+# the same for concurrent_compactors setting:
+export CONCURRENT_COMPACTORS=$ncores
```

Comment:
@lmineiro yes, we've seen extra cpu utilization while testing with increased number of concurrent compactors, but flush writers seem to be OK.  On the other hand, more flush writers mean smaller SSTables, which in turn means more compaction...

The reasoning to raise number of flush writers is given in https://github.com/zalando/planb-cassandra/issues/21  The idea is to reduce the amount of CommitLog to replay in an event of a crash/restore on a node.